Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -264,7 +237,6 @@ protocol PreplannedMapAreaProtocol {
     var title: String { get }
     var description: String { get }
     var thumbnail: LoadableImage? { get }
-    var id: PortalItem.ID? { get }
```

Comment:
This wasn't used anywhere from the protocol. Everywhere it was used there is an actual PreplannedMapArea instance.